source/embed.py

```diff
@@ -492,9 +492,9 @@ def embed_sentences(
 
     if encoder_path:
         encoder = load_model(
-            encoder_path,
-            spm_model,
-            bpe_codes,
+            str(encoder_path),
```

Comment:
Hi @julianpollmann! Thanks so much for the suggestions. Indeed the function `embed_sentences` accepts as input some `Path` parameters. However, instead of moving these to strings later on, I think ideally we should be more explicit in the function arguments. Instead of casting to string later on, perhaps we can try the following:

In the `embed_sentences` definition, move the type for `ifname`, `encoder_path`, `bpe_codes`, and `spm_model` to be of type `str`

What do you think?

Reply:
Yes, that's what I also thought, but didn't want to break the method signature. I'll update the PR.

Reply:
@julianpollmann great!!